lib/graph.js

```diff
@@ -459,32 +467,44 @@ Graph.prototype.removeEdge = function(v, w, name) {
   return this;
 };
 
-Graph.prototype.inEdges = function(v, u) {
+Graph.prototype.getInEdges = function (v, u) {
   var inV = this._in[v];
   if (inV) {
     var edges = _.values(inV);
     if (!u) {
       return edges;
     }
-    return _.filter(edges, function(edge) { return edge.v === u; });
+    return _.filter(edges, function (edge) {
+      return edge.v === u;
+    });
   }
 };
 
-Graph.prototype.outEdges = function(v, w) {
+Graph.prototype.inEdges = function (v, u) {
+  return this.isDirected() ? this.getInEdges(v, u) : this.nodeEdges(v, u);
+};
+
+Graph.prototype.getOutEdges = function (v, w) {
   var outV = this._out[v];
   if (outV) {
     var edges = _.values(outV);
     if (!w) {
       return edges;
     }
-    return _.filter(edges, function(edge) { return edge.w === w; });
+    return _.filter(edges, function (edge) {
+      return edge.w === w;
+    });
   }
 };
 
-Graph.prototype.nodeEdges = function(v, w) {
-  var inEdges = this.inEdges(v, w);
+Graph.prototype.outEdges = function (v, w) {
+  return this.isDirected() ? this.getOutEdges(v, w) : this.nodeEdges(v, w);
+};
+
+Graph.prototype.nodeEdges = function (v, w) {
+  var inEdges = this.getInEdges(v, w, false);
   if (inEdges) {
-    return inEdges.concat(this.outEdges(v, w));
+    return inEdges.concat(this.getOutEdges(v, w, false));
```

Comment:
As far as I can tell, `getInEdges` only has two arguments, `v` and optionally `w`, making the `false` seem useless here.